InformationScripting/InformationScripting.pro

```diff
@@ -34,56 +29,54 @@ HEADERS += src/precompiled.h \
     src/queries/SubstractOperator.h \
     src/queries/AddASTPropertiesAsTuples.h \
     src/nodes/TagExtension.h \
     src/queries/TagQuery.h \
-    src/nodes/QueryNode.h \
-    src/nodes/CommandNode.h \
-    src/nodes/CommandArgument.h \
-    src/nodes/CompositeQueryNode.h \
-    src/nodes/QueryNodeContainer.h \
-    src/visualization/VCommandNode.h \
-    src/visualization/VCommandArgument.h \
-    src/visualization/VCommandNodeStyle.h \
-    src/visualization/VCommandArgumentStyle.h \
-    src/handlers/HQuery.h \
-    src/parsing/QueryParser.h \
-    src/nodes/OperatorQueryNode.h \
-    src/visualization/VOperatorQueryNode.h \
-    src/visualization/VOperatorQueryNodeStyle.h \
     src/queries/LinearQuery.h \
-    src/visualization/HighlightOverlay.h \
-    src/visualization/Heatmap.h \
     src/queries/VersionControlQuery.h \
-    src/parsing/SimpleQueryParser.h \
-    src/visualization/VCompositeQueryNode.h \
-    src/visualization/VCompositeQueryNodeStyle.h \
-    src/misc/Optional.h \
-    src/parsing/ArgumentParser.h \
-    src/parsing/ArgumentRule.h \
-    src/parsing/QueryParsingException.h \
     src/queries/BreakpointManager.h \
     src/queries/RuntimeQuery.h \
-    src/misc/AstModification.h \
-    src/query_prompt/QueryPromptInput.h \
     src/query_prompt/QueryPromptMode.h \
-    src/query_prompt/QueryPromptInputStyle.h \
-    src/query_prompt/VQueryNodeContainer.h \
-    src/query_prompt/QueryBuilder.h \
-    src/queries/PassthroughQuery.h
+    src/queries/PassthroughQuery.h \
+    src/query_prompt/nodes/CommandArgument.h \
+    src/query_prompt/nodes/CommandNode.h \
+    src/query_prompt/nodes/CompositeQueryNode.h \
+    src/query_prompt/nodes/OperatorQueryNode.h \
+    src/query_prompt/nodes/QueryNode.h \
+    src/query_prompt/nodes/QueryNodeContainer.h \
+    src/query_prompt/visualization/VOperatorQueryNodeStyle.h \
+    src/query_prompt/visualization/VOperatorQueryNode.h \
+    src/query_prompt/visualization/VCompositeQueryNodeStyle.h \
+    src/query_prompt/visualization/VCompositeQueryNode.h \
+    src/query_prompt/visualization/VCommandNode.h \
+    src/query_prompt/visualization/VCommandNodeStyle.h \
+    src/query_prompt/visualization/VCommandArgumentStyle.h \
+    src/query_prompt/visualization/VCommandArgument.h \
+    src/query_prompt/parsing/QueryBuilder.h \
+    src/query_prompt/visualization/VQueryNodeContainer.h \
+    src/query_prompt/visualization/QueryPromptInputStyle.h \
+    src/query_prompt/visualization/QueryPromptInput.h \
+    src/query_prompt/HQuery.h \
+    src/python_bindings/AstApi.h \
+    src/python_bindings/DataApi.h \
+    src/python_bindings/BoostPythonHelpers.h \
+    src/python_bindings/PythonSet.h \
+    src/python_bindings/AstModification.h \
+    src/query_framework/Optional.h \
+    src/query_framework/QueryRegistry.h \
+    src/query_framework/QueryExecutor.h \
+    src/query_framework/ArgumentParser.h \
+    src/query_framework/ArgumentRule.h \
+    src/QueryParsingException.h \
```

Comment:
Hm I wasn't sure where to put it, since it is also used in query_prompt. But I guess that makes sense.